roles/commons/tasks/firewall.yml

```diff
@@ -24,6 +24,8 @@
     - /etc/firewalld/direct.xml
   tags: firewall
   notify: restart firewall
+  when: clear_firewall is defined
```

Comment:
I added this variable so that it doesn't delete the Firewall in each run of this task.

In our case, when we run the Icinga role we don't want to delete the previously defined firewall.